libraries/bot-integration-core/src/main/java/com/microsoft/bot/integration/CloudAdapter.java

```diff
@@ -0,0 +1,55 @@
+package com.microsoft.bot.integration;
```

Comment:
I missed the copyright notice. Have to add it

Reply:
Added